.dockerignore

```diff
@@ -1,4 +1,9 @@
-deps/*
-install/*
+/deps/*
+!/deps/readies/
```

Comment:
I've always thought about `deps`, `build` and `install` as a throw-away directories. I think we should keep `readies` and `system-setup.py` in a separate directory (like `util`)

Reply:
The new model to be introduced as part of the ARM work, will introduce the "bin/arch-os-variant" scheme to rule all binary/disposable files, and use "build" for different purpose (yielding it undisposable). "deps" will be used for submodules and the like (with readies eventually turning into a git submodule).
I currently keep system-setup.py at module root for all Redis modules for accessibility, but I do seriously consider a "scripts" directory. Yes, I do admit having a moderate directory structure OCD. :-)
For now, I'd like the model to settle down, let it run for a while, and then find its stable form.